assets/js/blocks/cart-checkout/mini-cart/block.tsx

```diff
@@ -128,16 +129,25 @@ const MiniCartBlock = ( {
 				} }
 				aria-label={ ariaLabel }
 			>
-				{ sprintf(
-					/* translators: %d is the count of items in the cart. */
-					_n(
-						'%d item',
-						'%d items',
-						cartItemsCount,
-						'woo-gutenberg-products-block'
-					),
-					cartItemsCount
-				) }
+				<span className="wc-block-mini-cart__amount">
+					{ formatPrice(
+						subTotal,
+						getCurrencyFromPriceResponse( cartTotals )
+					) }
+				</span>
+				<span className="wc-block-mini-cart__count">
+					{ sprintf(
+						/* translators: %d is the count of items in the cart. */
+						_n(
+							'%d item',
+							'%d items',
+							cartItemsCount,
+							'woo-gutenberg-products-block'
+						),
+						cartItemsCount
+					) }
+				</span>
+				<QuantityBadge count={ cartItems.length } />
```

Comment:
The `cartItemsCount` variable seems more descriptive. Is there a reason for not using it here?
```suggestion
				<QuantityBadge count={ cartItemsCount } />
```

Reply:
No, it's my mistake, thanks for the head-up!